storage/src/main/java/org/apache/kafka/storage/internals/log/CleanShutdownFileHandler.java

```diff
@@ -49,13 +47,15 @@ public class CleanShutdownFileHandler {
     private static final int CURRENT_VERSION = 0;
     private final Logger logger;
 
-    private enum Fields {
-        VERSION,
-        BROKER_EPOCH;
+    private static class Content {
+        public int version;
+        public Long brokerEpoch;
 
-        @Override
-        public String toString() {
-            return name().toLowerCase(Locale.ROOT);
+        public Content() {};
```

Comment:
Is this constructor used?

Reply:
This is the constructor used by ObjectMapper().readValue